api/jobs/jobs.py

```diff
@@ -162,6 +173,16 @@ def insert(self):
         result = config.db.jobs.insert_one(self.mongo())
         return result.inserted_id
 
+    def save(self):
+        self.modified = datetime.datetime.utcnow()
+        update = self.mongo()
+        config.log.debug(update)
+        job_id = update.pop('id')
+        result = config.db.jobs.replace_one({'_id': job_id}, update)
+        if result.modified_count != 1:
+            raise Exception('Job modification not saved')
+        return {'modified_count': 1}
```

Comment:
It's an existing practice across the API. Not married to the idea or anything, but might as well keep it consistent for now.